ui/src/app/utils/institutions.tsx

```diff
@@ -44,6 +44,10 @@ export const getRoleOptions = (institutions: Array<PublicInstitutionDetails>, in
     const {shortName} = institution;
     return shortName === institutionShortName;
   }, institutions);
+  if (matchedInstitution === null || matchedInstitution === undefined) {
```

Comment:
direct comparison to undefined makes me nervous. Is everything getting initialized correctly?

Reply:
This is a check of a call to `fp.find()` which returns undefined if it is not found.  I removed the null check since it's not relevant.